### ISSUE TITLE
Server Crash Caused by Provider Seal

### ISSUE THREAD

**Issue by Xirema** [Azanor/thaumcraft-5#313]
I've attached the Crash Report to this Issue. The long and short of it is that Provider Seals are causing a Null Pointer Exception under certain conditions.

The basic setup is that I've created a large underground area where I'm farming Silverwood Trees. There's a number of golems down there, some with precise hands, some with block-breaking hands. All made of greatwood, all have the sight + smart head types. Near the center of the room, there's 8 single-chests each with both a Store Seal and a Provider Seal. Each has a Comparator and a Redstone Lamp adjacent so that it's easily visible which chests have inventories in them. Around the room are 25 Use Seals set to target Silverwood Saplings when the block is empty. It requests provisioning. There are 16 each of Gather seals and Lumberjacking seals to provider coverage of the whole room. The gather seals are set to only gather Silverwood Saplings.

Basically, what I'm saying is, it's a perfect system that plants Silverwood Saplings, then cuts them down. Except for the part where I occasionally get a NullPointerException on the server.

I can provide any other needed information. Not sure what's needed at this time.

[crash-2016-01-22_21.59.36-server.txt](https://github.com/Azanor/thaumcraft-5/files/101706/crash-2016-01-22_21.59.36-server.txt)

**Comment by shadowbolt79** [comment]
#338, an issue I created earlier today, seems to be related to this. Azanor, feel free to close the other one since it is probably a duplicate. After watching them for a while, I have determined that the crash occurs the moment a golem pulls items out of a provide chest.
